src/_data/team.json

```diff
@@ -249,16 +249,6 @@
     "mastodon": "",
     "instagram": ""
   },
-  {
-    "name": "Saptak",
```

Comment:
@SaptakS fill this up and not remove it 😊 please!

Reply:
Updated....... 😅